contentcuration/contentcuration/viewsets/assessmentitem.py

```diff
@@ -142,7 +203,6 @@ class AssessmentItemViewSet(BulkCreateMixin, BulkUpdateMixin, ValuesViewset):
     filter_backends = (DjangoFilterBackend,)
     filter_class = AssessmentItemFilter
     values = (
-        "id",
```

Comment:
Will this cause issues if exercises have been copied and have the same assessment_id?

Reply:
Shouldn't do, as the assessments are keyed by `assessment_id` and `contentnode` jointly in the frontend.